mev_inspect/classifiers/specs/zero_ex.py

```diff
@@ -1,9 +1,97 @@
+from typing import Optional, List
+from mev_inspect.schemas.transfers import Transfer
+from mev_inspect.schemas.swaps import Swap
 from mev_inspect.schemas.traces import (
+    DecodedCallTrace,
     Protocol,
 )
 from mev_inspect.schemas.classifiers import (
     ClassifierSpec,
+    SwapClassifier,
 )
+from mev_inspect.classifiers.helpers import (
+    get_first_amount_transferred_to_address,
+    get_first_amount_transferred_by_token_address,
+)
+
+
+class ZeroExSwapClassifier(SwapClassifier):
+    @staticmethod
+    def parse_swap(
+        trace: DecodedCallTrace,
+        prior_transfers: List[Transfer],
+        child_transfers: List[Transfer],
+    ) -> Optional[Swap]:
+
+        ANY_TAKER = "0x0000000000000000000000000000000000000000"
```

Comment:
this isn't really an accurate name

can just call the `ZERO_ADDRESS`

most ideal would be for the code to read as the explanation

```
def is_any_taker(address: str) -> bool:
    return address == ZERO_ADDRESS
```